pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -600,9 +600,6 @@ def __resolve_ckpt_dir(self, trainer: "pl.Trainer") -> None:
 
         self.dirpath = ckpt_path
 
-        if not trainer.fast_dev_run and trainer.training_type_plugin.should_rank_save_checkpoint:
```

Comment:
to confirm, if the trainer is in fast_dev_run, there should be no checkpoint directory created?

Reply:
yep it'll just skip checkpointing (so IOPlugin makedirs) too https://github.com/PyTorchLightning/pytorch-lightning/blob/ed84cef3afa8db1381162cf86bf2992bce71f9fb/pytorch_lightning/callbacks/model_checkpoint.py#L383-L391